_RELEASE/Packs/experimental/Scripts/Levels/text_shader.lua

```diff
@@ -0,0 +1,20 @@
+function onInit()
+	text_shader = shdr_getShaderId("text.frag")
+
+	-- RenderStage 8 is text
+	shdr_setActiveFragmentShader(8, text_shader)
```

Comment:
There is an enum in `utils.lua` for this purpose, we should add the render stage there

```suggestion
u_execDependencyScript("ohvrvanilla", "base", "vittorio romeo", "utils.lua")

-- ...

	-- RenderStage 8 is text
	shdr_setActiveFragmentShader(RenderStage.TEXT, text_shader)
```